Makefile

```diff
@@ -47,7 +47,11 @@ db: python
 
 .PHONY: dev
 dev: build/manifest.json python
-	@tox -qe dev
+	@supervisord -c conf/supervisord-dev.conf
+
+.PHONY: devssl
+devssl: build/manifest.json python
+	@env H_GUNICORN_CERTFILE=.tlscert.pem H_GUNICORN_KEYFILE=.tlskey.pem APP_URL='https://localhost:5000' WEBSOCKET_URL='wss://localhost:5001/ws' supervisord -c conf/supervisord-dev.conf
```

Comment:
`make devssl` replaces [running `hypothesis devserver --https`](https://h.readthedocs.io/en/latest/developing/ssl/) (and the docs are updated)